examples/src/fibonacci/fib2/mod.rs

```diff
@@ -29,21 +30,26 @@ const TRACE_WIDTH: usize = 2;
 // FIBONACCI EXAMPLE
 // ================================================================================================
 
-pub fn get_example(options: ExampleOptions, sequence_length: usize) -> Box<dyn Example> {
-    Box::new(FibExample::new(
-        sequence_length,
-        options.to_proof_options(28, 8),
-    ))
+pub fn get_example(options: &ExampleOptions, sequence_length: usize) -> Box<dyn Example> {
```

Comment:
We might need to return a `Result` from here. This would be useful in cases when a given example does not support the requested hash function.

Reply:
Make sense